x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -12,6 +12,7 @@ import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 // @ts-ignore
```

Comment:
Unrelated to this PR but I think we can remove this `// @ts-ignore`

Reply:
Fixed in [76794bd](https://github.com/elastic/kibana/pull/132963/commits/76794bd51023db9215479fc5debdd65c19a89b87).